system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -17,12 +17,16 @@
 
 #include <chrono>
 #include <string>
+#include <stdexcept>
```

Comment:
Alphasort

Reply:
Hmm, strange that the linters didn't catch this.

Reply:
Done.